bin/mdsctl

```diff
@@ -333,7 +337,13 @@ templateDockerCompose() {
 
 # TODO: DRY
 templateValues() {
-  suffix=$(getBranchSuffix)
+  branchSuffix=$(getBranchSuffix)
+  if [ "$branchSuffix" = "-master" ]; then
+    suffix="$branchSuffix"
+  else
+    suffix="$branchSuffix"$(getCommitHash)
```

Comment:
You could insert the `-` here instead of using `sed` in `getCommitHash` since the `-` isn't actually part of the hash.

Reply:
Yeah, I was just following the pattern from `getBranchSuffix()`. I'm happy with either!